jiant/models.py

```diff
@@ -746,6 +761,12 @@ def build_lm(task, d_inp, args):
     return hid2voc
 
 
+def build_mlm(task, d_inp, task_params, embedder):
+    " Build MLM components " ""
+    lm_head = embedder.get_pretrained_lm_head()
```

Comment:
if build mlm is simply getting pretrained head, why does it need task, d_inp, task_params as argument as well?

Reply:
Good point, I'll delete those.